static/templates/actions_popover_content.hbs

```diff
@@ -96,6 +96,12 @@
     </li>
     {{/if}}
 
+    <li>
+        <a class="view_read_receipts" data-message-id="{{message_id}}" tabindex="0">
+            <i class="zulip-icon zulip-icon-readreceipts" aria-label="{{#tr}}View read receipts{{/tr}}"></i> {{t "View read receipts" }}
+        </a>
+    </li>
```

Comment:
This should be hidden for spectators, who have no access to the relevant API endpoint. I think this can be done by just adding `class="hidden-for-spectators"` to the `li` element.

Reply:
Added this class.